lib/liquidTestRunner.js

```diff
@@ -409,9 +413,12 @@ async function runTestsWithOutput(
 // RETURN (AND LOG) ONLY PASSED OR FAILED
 // CAN BE USED BY GITHUB ACTIONS
 async function runTestsStatusOnly(firmId, handle, testName = "") {
-  const testRun = await runTests(firmId, handle, testName, false, "none");
```

Comment:
We can actually only change this row to:

```javascript
  const testRuns = await runTests(firmId, handle, testName, false, "none");
  const testRun = testRuns.testRun;
```

And I think that should be all to work again